src/main/webapp/js/user-page-loader.js

```diff
@@ -184,7 +184,7 @@ function fetchAboutMe() {
  * @param {Message} message
  * @return {Element}
  */
-function buildMessageDiv(message) {
+function buildMessageDiv(message, margin) {
```

Comment:
Should probably update the comment of this to include the new param and/or an explanation of how the function works

Reply:
Will do!